velox/common/base/SimdUtil.h

```diff
@@ -497,12 +496,20 @@ inline void transpose(
   }
   if (i < size) {
     const auto numLeft = size - i;
+#if XSIMD_WITH_AVX2
     auto mask = simd::leadingMask<TData>(numLeft);
     auto indexBatch = loadGatherIndices<TData, TIndex>(indices.data() + i);
     const auto values = simd::maskGather<TData, TIndex>(
         xsimd::broadcast<TData>(0), mask, input, indexBatch);
     storeLeading<TData, A>(values, mask, numLeft, output + i);
+#else
+    #pragma unroll
+    for (uint32_t k = 0; k < numLeft; ++k) {
+	register uint32_t offset = i + k;
```

Comment:
```suggestion
	uint32_t offset = i + k;
```
it's doing nothing, see https://en.cppreference.com/w/cpp/keyword/register.html